AK/Weakable.h

```diff
@@ -66,7 +66,7 @@ class WeakLink : public RefCounted<WeakLink> {
 
     bool is_null() const
     {
-        return !unsafe_ptr<void>();
+        return nullptr == unsafe_ptr<void>();
```

Comment:
Although I like Yoda conditions, I think it's closer to "the common project style" to write `unsafe_ptr<void>() == nullptr`.